lib/saml_idp/encryptor.rb

```diff
@@ -61,7 +61,6 @@ def build_encryption_template
           key_info.EncryptedKey Id: 'EK', xmlns: 'http://www.w3.org/2001/04/xmlenc#' do |enc_key|
             enc_key.EncryptionMethod Algorithm: key_transport_ns
             enc_key.tag! 'ds:KeyInfo', 'xmlns:ds' => 'http://www.w3.org/2000/09/xmldsig#' do |key_info2|
-              key_info2.tag! 'ds:KeyName'
```

Comment:
As far I understand `KeyInfo` follows [XML Signature Syntax ](https://www.w3.org/TR/xmldsig-core2/#sec-KeyInfo) this one mostly used when you have different keys for `signing` and `encryption`. But this gem doesn't have feature to can specify different keys. So, I think this won't break anything.

Reply:
Gotcha thanks for the explanation. Perhaps something for folks to consider in the future.